spec/coder_spec.lua

```diff
@@ -1082,6 +1082,30 @@ describe("Pallene coder /", function()
         end)
     end)
 
+    describe("sqrt builtin", function()
+        setup(compile([[
+            function square_root(x: float): float
+                return sqrt(x)
+            end
+        ]]))
+
+        it("works on positive numbers", function()
+            run_test([[
+                assert(1.0 == test.square_root(1.0))
```

Comment:
It might be nice to test some corner cases such as nan, inf and negative numbers.

Reply:
Added NaN and inf. We already had tests for negative numbers.